tests/test_views.py

```diff
@@ -105,28 +106,38 @@ def test_review_success_all_files(test_outputs):
             assert actual_path.read_bytes() == zip_obj.open(zip_path).read()
 
 
-def test_review_success_no_files(test_outputs):
-    request = get_review_url_request(test_outputs)
-    response = views.review(request)
+def test_release_success_no_files(test_outputs):
+    request = get_release_url_request(test_outputs)
+    response = views.release(request)
     assert response.status_code == 400
 
 
-def test_review_success_unrecognized_files(test_outputs):
-    request = get_review_url_request(test_outputs, outputs=["output-does-not-exist"])
-    response = views.review(request)
+def test_release_success_unrecognized_files(test_outputs):
+    request = get_release_url_request(test_outputs, outputs=["output-does-not-exist"])
+    response = views.release(request)
     assert response.status_code == 400
 
 
-def test_review_missing_metadata(tmp_path):
+def test_release_missing_metadata(tmp_path):
     path = tmp_path / "results.json"
     path.write_text(json.dumps({"test": {"output": "does-not-exist"}}))
-    url = RequestFactory().get("/review", data={"path": str(path)}).get_full_path()
+    url = RequestFactory().get("/release", data={"path": str(path)}).get_full_path()
     request = RequestFactory().post(url, data={"outputs": ["test"]})
-    response = views.review(request)
+    response = views.release(request)
     zf = io.BytesIO(response.getvalue())
     with zipfile.ZipFile(zf, "r") as zip_obj:
         assert zip_obj.testzip() is None
         assert zip_obj.namelist() == ["results.json", "missing-files.txt"]
         contents = zip_obj.open("missing-files.txt").read().decode("utf8")
         assert "were not found" in contents
         assert "does-not-exist" in contents
+
+
+def test_release_success_logs_audit_trail(test_outputs, mocker):
+    mocked_local_audit = mocker.patch("sacro.views.local_audit")
```

Comment:
I am not convinced its worth pulling in an entire dependency just for this test to use the mocker fixture.

We could use pytest's builtin `monkeypatch`, or my preference would be to assert the log output is actually generated, like in other tests, rather than the function is called?

Reply:
Ah, I keep forgetting about `monkeypatch`. Those are good suggestions, thank you.